.github/actions/e2e_mini/run-e2e.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# This script installs all dependencies for MiniConstellation to be run on a
+# fresh Ubuntu 22.04 LTS installation.
+# It expects to find the to be used Constellation CLI to be available at
+# $HOME/constellation
+#
+set -euxo pipefail
+
+cloud-init status --wait
+
+export DEBIAN_FRONTEND=noninteractive
+sudo apt update -y
+
+sudo apt install -y bridge-utils cpu-checker libvirt-clients libvirt-daemon libvirt-daemon-system qemu qemu-kvm virtinst xsltproc ca-certificates curl gnupg lsb-release
+
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update -y
+sudo apt install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
```

Comment:
nit: Maybe put a output group around the apt operations, makes the test output easier to read imo